tests/unit/Micro/Plugin/Http/Business/Locator/RouteLocatorFactoryTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ *  This file is part of the Micro framework package.
+ *
+ *  (c) Stanislau Komar <kost@micro-php.net>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace Micro\Plugin\Http\Business\Locator;
```

Comment:
@Asisyas, it is better to add `Tests` to namespaces in tests files so test classes are not in the core namespace.
```suggestion
namespace Micro\Plugin\Http\Tests\Business\Locator;
```